addons/point_of_sale/static/src/app/debug/debug_widget.js

```diff
@@ -158,7 +158,7 @@ export class DebugWidget extends Component {
                 for (const rel of manyRel) {
                     const model = this.pos.models[rel.relation];
 
-                    if (!model) {
+                    if (!model || !order[rel.name] || !order[rel.name].length) {
```

Comment:
I know showing a traceback is not the best but couldn't this lead to imported orders not having some important fields and leading to traceback afterwards ?

Reply:
I don't understand how this can be possible?

Does this mean that if someone tries to export an order with x modules and import it into another database with y modules?

Reply:
@davidmonnom No, it doesn't work if you export an order from a database and importing to it again. Because some fields are missing from the exported order.

Reply:
@davidmonnom Could you please review this pr? Currently it's not possible to import the exported orders.

Reply:
I'm not convinced by the fix to be honest. it seems to come more from the imported data. Not really the import. How can fields be missing? And which ones?